### DISCUSSION TITLE
A question about Distrobox functionality

### DISCUSSION THREAD

**Discussion by Avanesov79** [89luca89/distrobox#1059] [General]
Is there a way to:
1. Make Distrobox initialize on creation with distrobox-assemble so I dont need to manually enter it?
2. Use the container.ini assemble file to run distrobox-export on creation

My goal is to try and setup a .ini file for distrobox-assemble and have it both start the container and link my specified applications into my host system. I just dont know if this is possible and im overlooking the relevant commands or not.

**Comment by 89luca89** [comment]
Hi @Avanesov79 

for n1, check the docs, there is a `start_now=true` option to start the container immediately
for n2, check the docs, there are options to export apps/bins during `assemble create`

**Reply by Avanesov79** [reply]
Ok, I see the start-now option but I do not see anything obvious for exporting apps from distrobox-assemble.

On https://distrobox.it/usage/distrobox-assemble/ under "Available options" there doesnt seem to be anything to do that. Am I looking in the wrong place?

And thanks for the prompt response.

**Reply by 89luca89** [reply]
Right, that was not clear from me, sorry

With latest git version (so you need the `--next` option in the curl/wget installer) you can do that
As explained in the git documentation here: https://github.com/89luca89/distrobox/blob/main/docs/usage/distrobox-assemble.md

**Reply by Avanesov79** [reply]
Awesome, Thank you very much.
